src/ds/ds-device-common.cpp

```diff
@@ -256,6 +257,13 @@ namespace librealsense
 
     void ds_device_common::update_flash(const std::vector<uint8_t>& image, rs2_update_progress_callback_sptr callback, int update_mode)
     {
+        // check if the given FW size matches the expected FW size
+        if( image.size() != unsigned_fw_size )
+        throw librealsense::invalid_value_exception( rsutils::string::from()
```

Comment:
indent still looks off

Reply:
Oh, now I see the indent issue, I fixed the comment indent before.
Fixed